daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf2.proto

```diff
@@ -16,6 +16,7 @@
 
 
 // Minor version history:
+// * 2.1 --  2024-08-03: Initial version
```

Comment:
```suggestion
// * 2.1 --  2024-03-18: Initial version
```

Reply:
that is, unless you actually meant August 3rd

Reply:
Good catch.